dbt/include/global_project/macros/adapters/bigquery.sql

```diff
@@ -11,11 +11,27 @@
 {%- endmacro -%}
 
 
+{% macro cluster_by(raw_cluster_by) %}
+  {%- if raw_cluster_by is none -%}
+    {{ return('') }}
+  {% endif %}
+
+  {% set cluster_by_clause %}
+    cluster by {{ raw_cluster_by }}
```

Comment:
i think this is supposed to be a CSV of column names. That means that folks would do something like

```
{{
  config(
    materialized='table',
    cluster_by='field_1, field_2'
  )
}}
```

I think that's reasonable, but we accept an actual `list` of values in other similar configs: https://github.com/fishtown-analytics/dbt/blob/development/dbt/include/global_project/macros/adapters/redshift.sql#L21

all things being equal, I think it would be good to keep this interface with other configs like this!